inventory.yaml

```diff
@@ -7,26 +7,47 @@ images:
       context: .
       template_context: scripts/dev/templates/agent
 
+    inputs:
+      - agent_version
+      - tools_version
+
     stages:
+      - name: agent-ubuntu-context
+        task_type: docker_build
+        dockerfile: scripts/dev/templates/agent/Dockerfile.builder
+        tags: ["ubuntu"]
+        buildargs:
+          agent_version: $(inputs.params.agent_version)
+          tools_version: $(inputs.params.tools_version)
+          agent_distro: linux_x86_64
+          tools_distro: ubuntu1604-x86_64
+
+        output:
+          - registry: $(inputs.params.registry)/agent-ubuntu-context-dev
+            tag: $(inputs.params.version_id)
+
       - name: agent-template-ubuntu
         task_type: dockerfile_template
-        distro: ubuntu
         tags: ["ubuntu"]
+        distro: ubuntu
 
+        # TODO: remove - this is only here as sonar raises a key error on version_id
+        # if no other inputs are specified.
         inputs:
-          - agent_version
-          - tools_version
-          - tools_distro
-          - agent_distro
+          - noop
```

Comment:
I think you fixed this already :)

Reply:
we haven't yet released 0.10.0, but let's do that now, no harm!